charts/opentelemetry-demo-datadogv2/agent-ingest.yaml

```diff
@@ -39,7 +39,16 @@ opentelemetry-demo:
           value: "true"
     featureflagService: *override
     ffsPostgres: *override
-    frontend: *override
+    frontend:
+      envOverrides:
+        - name: OTEL_EXPORTER_OTLP_ENDPOINT
+          value: 'http://$(HOST_IP):$(OTLP_PORT_1)'
+        - name: LOCUST_HEADLESS
+          value: "true"
+        - name: DD_RUM_APPLICATION_ID
+          value: "XX"
```

Comment:
Are the values for DD_RUM_APPLICATION_ID and DD_RUM_CLIENT_TOKEN required to install and run the demo chart? If so we might want to document how to populate these fields